tests/ice40/run-test.sh

```diff
@@ -0,0 +1,33 @@
+set -e
+if [ -f "../../techlibs/common/simcells.v" ]; then
+	COMMON_PREFIX=../../techlibs/common
+	TECHLIBS_PREFIX=../../techlibs
+else
+	COMMON_PREFIX=/usr/local/share/yosys
+	TECHLIBS_PREFIX=/usr/local/share/yosys
+fi
+{
+echo "all::"
+for x in *.ys; do
+	echo "all:: run-$x"
+	echo "run-$x:"
+	echo "	@echo 'Running $x..'"
+	echo "	@../../yosys -ql ${x%.ys}.log $x -w 'Yosys has only limited support for tri-state logic at the moment.'"
+done
+for t in *_tb.v; do
+	echo "all:: run-$t"
+	echo "run-$t:"
```

Comment:
change this line to :
echo "run-$t: ${t%_tb.v}_synth.v"

Reply:
done